app/models/verification_attempt.rb

```diff
@@ -44,8 +47,8 @@ def state_machine
   def attachment_file_types
     [selfie, photo_identification].each do |attachment|
       if attachment.present?
-        unless FileTypeAllowedValidator::VALID_MIME_TYPES.include?(attachment.content_type)
-          errors.add(attachment.name, I18n.t("validators.file_type"))
+        unless VALID_MIME_TYPES.include?(attachment.content_type)
+          errors.add(attachment.name, I18n.t("validators.file_type", valid_types: VALID_FILE_TYPES.to_sentence(locale: I18n.locale)))
```

Comment:
Because of `switch_locale` in `ApplicationController`

```
class ApplicationController < ActionController::Base
# ...
  around_action :switch_locale
```

we probably don't need the `locale: I18n.locale` here. My request is that you manually test this by switching to an ES locale on the website with the `locale: I18n.locale` code removed.